pinecone/sparse/bm25/bm25-vector-generation.ipynb

```diff
@@ -0,0 +1,6251 @@
+{
```

Comment:
<p>"It is (a) simple but effective..."</p><p>"In the following guide, ..."</p><p>"sparse-dense index" -&gt; "sparse-dense vectors" (There's no specific index for sparse-dense)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>